doc/source/getting_started/running_mapdl.rst

```diff
@@ -173,6 +173,63 @@ the output of MAPDL within Python and can be used to debug why MAPDL
 isn't launching.  Output will be limited on Windows due to the way
 MAPDL launches on Windows.
 
+Default executable Location
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The first time you run PyMAPDL it will detect the
+available ANSYS installations, normally under:
+
+.. code:: text
+
+    C:/Program Files/ANSYS Inc/vXXX
+
+in windows, and:
+
+.. code:: text
+
+    /usr/ansys_inc/vXXX
+
+on Linux.
+
+If PyMAPDL successfully find a valid ANSYS installation, it will cache its
+path in the configuration file ``config.txt`` which can be found by following
+the next code:
+
+.. code:: python
+
+    >>> from ansys.mapdl.core.launcher import CONFIG_FILE
+    >>> print(CONFIG_FILE)
+    'C:\\Users\\user\\AppData\\Local\\ansys_mapdl_core\\ansys_mapdl_core\\config.txt'
+
+
+In certain cases, this configuration might become obsolete. For example, when a new
+ANSYS version is installed, and the old one is removed.
+To update this configuration file with the latest path you can the following:
+
+.. code:: python
+
+    >>> from ansys.mapdl.core import save_ansys_path
+    >>> save_ansys_path(r"C:\Program Files\ANSYS Inc\v222\ansys\bin\winx64\ansys222.exe")
+    'C:\\Program Files\\ANSYS Inc\\v222\\ansys\\bin\\winx64\\ansys222.exe'
+
+If you want to check which ANSYS installations is PyMAPDL detecting you can issue:
+
+.. code:: python
+
+    >>> from ansys.mapdl.core.launcher import _get_available_base_ansys
+    >>> _get_available_base_ansys()
+    {222: 'C:\\Program Files\\ANSYS Inc\\v222',
+    212: 'C:\\Program Files\\ANSYS Inc\\v212',
+    -222: 'C:\\Program Files\\ANSYS Inc\\ANSYS Student\\v222'}
+
+It should be noticed that the student versions are provided as negative version (Python ``dict``
+does not accept two equal keys.
+The result of ``_get_available_base_ansys()`` is in order, higher version first and student versions
+at last.
```

Comment:
```suggestion
Student versions are provided as negative versions because the Python dictionary
does not accept two equal keys. The result of the ``_get_available_base_ansys()`` method
listing higher versions first and student versions last.
```